private_repo/ansible/inventory/dev/Core/hosts

```diff
@@ -162,6 +162,9 @@ local
 [log-forwarder:children]
 swarm-manager
 
+[dp-druid-broker]
+localhost                # Druid broker IP for druid proxy api
```

Comment:
please change this from localhost and give some dummy ip. It'll be confusing for adopters